test/parallel/test-throw-undefined-or-null-traced.mjs

```diff
@@ -0,0 +1,21 @@
+import { spawnPromisified } from '../common/index.mjs';
+import assert from 'node:assert';
+import { execPath } from 'node:process';
+import { describe, it } from 'node:test';
+
+
+describe('--trace-uncaught', () => {
+  it('prints a trace on process exit for uncaught errors', async () => {
+    for (const value of [null, undefined]) {
+      const { code, signal, stderr } = await spawnPromisified(execPath, [
+        '--trace-uncaught',
+        '--eval',
+        `throw ${value};`,
+      ]);
+      assert.match(stderr, /^Thrown at:$/m);
```

Comment:
consider using [`assret.snapshot()`](https://github.com/nodejs/node/pull/44095)

Reply:
The stack trace contains unstable parts (absolute paths, line numbers), so we can't just snapshot the values.

Reply:
👍🏻 we can `assert.snapshot(stabilize(trace))` just like currently python does in `message.py`, I was planning on doing that regardless of this pr, so I might just revisit it after this lands.

Reply:
@MoLow do you mind doing that in a follow up? I would love to get rid of the message/out tests altogether, as they're a pain to debug and don't follow the pattern of all the other tests.

Reply:
Sure will do